games/n.yaml

```diff
@@ -13,6 +13,22 @@
   repo: https://github.com/bobbens/naev
   type: clone
   url: http://naev.org/
+  
+- name: NBlood
+  type: remake
+  originals:
+  - Blood
+  repo: https://github.com/nukeykt/NBlood
+  url: https://nukeykt.retrohost.net/
+  development: active
+  status: playable
+  lang:
+  - C++
+  license:
+  - GPL
```

Comment:
```suggestion
  - GPL2
```

Reply:
My bad! I checked the EDuke32 license on the frontpage. They say the project using GPL license, but the link is for the GPL2 license.